browser/resources/settings/brave_settings_overrides.js

```diff
@@ -1,9 +1,331 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at http://mozilla.org/MPL/2.0/.
 
-window.BravePatching.registerPolymerBehaviors({
+// Local Scope
+(function() {
```

Comment:
does it make sense to break this file up at all? It's not terrible now, but seems like it could continue to grow and better to set a pattern of splitting into smaller files if that makes sense

Reply:
Yes. I was considering doing that in a follow-up with either the current html-modules approach, or invoking our webpack build target type to combine some js 'modules'. Since this PR can't land until C74 has re-landed, I may have some time to do that in this PR.